charts/airbyte/Chart.yaml

```diff
@@ -37,3 +38,27 @@ dependencies:
     name: minio
     version: 11.x.x
     repository: https://charts.bitnami.com/bitnami
+  - condition: airbyte-bootloader.enabled
+    name: airbyte-bootloader
+    repository: "https://airbytehq.github.io/helm-charts/"
+    version: 0.43.3
+  - condition: temporal.enabled
+    name: temporal
+    repository: "https://airbytehq.github.io/helm-charts/"
+    version: 0.43.3
+  - condition: webapp.enabled
+    name: webapp
+    repository: "https://airbytehq.github.io/helm-charts/"
+    version: 0.43.3
```

Comment:
curious, where did this version come from?

Reply:
It's a placeholder, it'll be replaced by mkver

Reply:
replaced with placeholder